useful_resources/sample_code/opt_outs_everyaction.py

```diff
@@ -0,0 +1,163 @@
+import os
+import ast
+import requests
+import time
+from parsons import Redshift, Table, VAN
+from parsons import logger
+from datetime import datetime
+
+# Credentials
+
+# The API_KEYS_STR variable contains API keys for EA committees in this format:
```

Comment:
Where is this variable coming from? Is this format how EA provides the data?

Reply:
@shaunagm Finally getting around to cleaning up this PR! This is not how EA provides the data. This format was developed by TMC to make it easier for scripts to update data in multiple committees. Some organizations have a parent-child committee structure and may have dozens of committees, but they will have a separate API key for each one.